.gitignore

```diff
@@ -29,3 +30,8 @@
 # other people from accidentally committing them.
 /package-lock.json
 /pnpm-lock.yaml
+
+# Python
+venv
+__pycache__
+.mypy_cache
```

Comment:
Hi @macfarlandian, can you move these into `backend/.gitignore`?

Reply:
oops too late but yes I will